main.go

```diff
@@ -29,9 +30,21 @@ type knownIssuesWorkflowInputSchema struct {
 }
 
 func openLinkInBrowser(link string) {
-	color.Green(fmt.Sprintf("Your new ticket has been successfully created! The link is %s \nWe will now try to open the new ticket for you in the browser. (Probably doesn't work on Windows.)", link))
-	cmd := exec.Command("open", link)
-	cmd.Run()
+	color.Green(fmt.Sprintf("Your new ticket has been successfully created! The link is %s \nWe will now try to open the new ticket for you in the browser.", link))
+	os := runtime.GOOS
+	switch os {
+	case "windows":
+		cmd := exec.Command("start", link)
+		cmd.Run()
+	case "darwin":
+		cmd := exec.Command("open", link)
+		cmd.Run()
+	case "linux":
+		cmd := exec.Command("xdg-open", link)
```

Comment:
Could you check for linux first that xdg-open is installed? Not all distributions have it available directly.

Reply:
Done, please re-review